control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -605,6 +605,8 @@ func (r *reconciler) reconcileConfig(ctx context.Context, hcp *hyperv1.HostedCon
 		errs = append(errs, fmt.Errorf("failed to reconcile network config: %w", err))
 	}
 
+	// copy proxy trustedCA to guest cluster
```

Comment:
nit: comment should start with Uppercase and finishes with period -> `Copy proxy trustedCA to guest cluster.`